src/readers/CMakeLists.txt

```diff
@@ -192,6 +192,16 @@ if(F3D_MODULE_ASSIMP)
   )
 endif()
 
+if(F3D_MODULE_ALEMBIC)
+  declare_f3d_reader(
+    NAME Alembic
+    EXTENSIONS abc
+    MIMETYPES application/vnd.abc
```

Comment:
this mime types is non standard, so a `mime-types-alembic-formats.xml` should be added

Reply:
Done